CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v1.3.1 / 2018-04-12
+
+* [CHANGE] Use alpine instead of scratch as base image.
```

Comment:
I understand that this has no connection with kms code base but should this be really included in a bugfix release?

Reply:
Fair point. I’ll close this and cherry-pick the actual fixes.